client/account/account.go

```diff
@@ -3,7 +3,7 @@ package account
 import (
 	"fmt"
 
-	"github.com/decred/dcrd/dcrec/secp256k1"
+	"github.com/decred/dcrd/dcrec/secp256k1/v2"
```

Comment:
This correction is unrelated to this PR, but should I scrub the go.sums of v1 entries?

Reply:
You could do a go mod tidy, but the module files are pretty much useless at this phase.